web/sites/default/config/extlink.settings.yml

```diff
@@ -0,0 +1,18 @@
+extlink_target: true
```

Comment:
I do not think we should force external links to open in a new window because it's not user friendly. I've provided more info about this in a comment on the Jira story.